src/main/webapp/scripts/toggle.js

```diff
@@ -24,11 +24,6 @@ async function loadContentSection() {
     locationObj = JSON.parse(getCookie('location'))
   }
   const keywords = await loadKeywords(KEYWORDS_OBJ_URL);
-  if (keywords == null) {
-    $('#login-error').removeClass('hide');
-    hideLoading();
-    return;
-  }
```

Comment:
I'm wondering whether you removed this. Had this in to initially block results page if the user isn't logged in.

Reply:
That's part of a different PR, where I'm trying to solve a bug